.github/workflows/ci.yaml

```diff
@@ -170,19 +170,14 @@ jobs:
 
           if [ "${{ matrix.environment }}" = "staging" ]; then
             SANDBOX_ID="STAGING_SANDBOX_ID"
+            CHART_REF="STAGING_CHART_REF"
           fi
           
-          if [ "${{ github.ref }}" = "refs/heads/master"]; then
-            CHART_REF="$GITHUB_SHA"
-          else
-            CHART_REF="$GITHUB_REF"
-          fi
-
-          echo "::set-output name=sandbox-id::$SANDBOX_ID"
```

Comment:
The idea is the same as with the sandbox ID. If we are deploying to a staging candidate file, we want the chart reference to be an easy value for the script to overwrite, so the user can configure the deployment to automatically update the chart or keep it at the latest version always.